src/ibex_bluesky_core/fitting.py

```diff
@@ -464,13 +463,31 @@ def guess(
         def guess(
             x: npt.NDArray[np.float64], y: npt.NDArray[np.float64]
         ) -> dict[str, lmfit.Parameter]:
+            c = np.mean(x)
+            scale = (np.max(y) - np.min(y)) / 2
+            b = np.min(y)
+
+            dy = np.max(y) - np.min(y)
+            y05 = np.min(y) + 0.05 * dy
+            y95 = np.min(y) + 0.95 * dy
+
+            ind05 = np.argmin(np.abs(y - y05))
+            ind95 = np.argmin(np.abs(y - y95))
+
+            x05 = x[ind05]
+            x95 = x[ind95]
+
+            erfc_const = 3  # The plotted erfc function where the greatest change
+            # in y happens in the region -1.5 and 1.5
```

Comment:
Please can we justify this constant mathematically?

If I follow naive logic with percentiles, this should be the difference between `erfinv(-0.9)` (aka -1.163...), and `erfinv(0.9)` (1.163), but that gives me 2.326 - not 3 - have you rounded that to 3? Or is there more maths I'm missing here?